.github/workflows/lra-test.yml

```diff
@@ -32,7 +32,8 @@ jobs:
         run: |
           git config --global user.email "you@example.com"
           git config --global user.name "Your Name"
-          WORKSPACE=$PWD PROFILE=LRA ./scripts/hudson/narayana.sh
+          WORKSPACE=$PWD PROFILE=LRA_THORNTAIL ./scripts/hudson/narayana.sh
```

Comment:
We need to migrate away from thorntail, for example I removed thorntail in favour of quarkus from the quickstarts because it was misbehaving, since we now have full support for running with quarkus can we use that instead.

In the lra/test pom `arq.thorntail` profile there's a comment saying `making the Quarkus coordinator to be started` which indicates we were once testing with quarkus.

What is the difference between ARQ_LRA_PROF and ARQ_PROF, why do we need an LRA specific variable, rts, xts, tomcat etc all use ARQ_PROF, what makes ARQ_LRA_PROF special?

Reply:
I think we are still running lra-coordinator on Quarkus. Thorntail is used to create the container that hosts lra-participants. Do not you think I should open a new JIRA to track the migration from Thorntail to Quarkus? @mmusgrov

I agree with you about ARQ_PROF. What do you think about this @ochaloup?

Reply:
@jmfinelli yes, the `ARQ_PROF` is fine to be jused. Agree with Mike that the LRA is not special and using the same variable name for all axes makes very good sense.

Quarkus/Thorntain: We need to move from Thorntail to Quarkus but we need a way how to do so. Quarkus needs to pass the TCKs.

At time I was experimenting with it the Quarkus was not able to work with the LRA. As far as I can say we need the Quarkus  extension. Before spending more time to have Quarkus running as the LRA client without extension I was thinking to wait until the LRA extension is developed (https://github.com/quarkusio/quarkus/pull/17903).
(plus, the part of the LRA spec are the non-JAX-RS participants that was the main trouble before - without the extension)

Reply:
@ochaloup I think you are overestimating the capabilities of the quarkus lra extension. Basically, all it does is pull in the correct dependencies (microprofile-lra-api and narayana-lra), adds a config property for the location of the coordinator, and adds some excludes from the application. 

Using quarkus to build LRA capable apps without the extension is pretty simple.

The blocker is that quarkus does not support CDI extensions (ie it's not the lra extension that's causing the issue).
When I wrote the initial implementation of non JAX-RS support I didn't need CDI at all (CDI was only added later because I had lost control over the implementation changes). I will investigate workarounds or I will look to removing the narayana LRA dependency on CDI extensions.

Reply:
It looks like the easiest (recommended) solution is to port the CDI extension code and I am looking into that. I will provide an update if/when I have more ...

Reply:
@mmusgrov yeap, agree. I wrote it wrongly - the trouble is to transform the CDI extension to make Quarkus working with LRA CDI.

Reply:
I will resolve the conversation.

Reply:
@mmusgrov @ochaloup That is what I indent to look into once Mike's PR is merged in Quarkus. The CDI extension we have is fairly simple functionality which can be rewritten in quarkus to be done during build time phase so we don't need to rely on CDI extension to do this processing. However, the CDI extension will be required in WFLY.